Gulpfile.js

```diff
@@ -133,6 +133,14 @@ function buildRollup(packages, targetBrowsers) {
       const bundle = await rollup.rollup({
         input,
         external,
+        onwarn(warning, warn) {
+          warn(warning);
```

Comment:
Can we move this inside the `if`? Circular dependencies are usually safe with ESM due to live bindings, which are emulated perfectly by rollup.